generators/cloudfoundry/index.js

```diff
@@ -46,38 +46,38 @@ module.exports = CloudFoundryGenerator.extend({
             message: 'Name to deploy as:',
             default: this.baseName
         },
-        {
-            type: 'list',
-            name: 'cloudfoundryProfile',
-            message: 'Which profile would you like to use?',
-            choices: [
-                {
-                    value: 'dev',
-                    name: 'dev'
-                },
-                {
-                    value: 'prod',
-                    name: 'prod'
-                }
-            ],
-            default: 0
-        },
-        {
-            when: function(response) {
-                return databaseType != 'no';
+            {
```

Comment:
I cancelled the build

Thanks & Regards,
Deepu

On Thu, Mar 3, 2016 at 8:41 PM, Vivek <notifications@github.com> wrote:

> In generators/cloudfoundry/index.js
> <https://github.com/jhipster/generator-jhipster/pull/3068#discussion_r54874114>
> :
>
> > -            choices: [
> > -                {
> > -                    value: 'dev',
> > -                    name: 'dev'
> > -                },
> > -                {
> > -                    value: 'prod',
> > -                    name: 'prod'
> > -                }
> > -            ],
> > -            default: 0
> > -        },
> > -        {
> > -            when: function(response) {
> > -                return databaseType != 'no';
> > +            {
>
> Hmm. Strange!
> Honestly all I did was get IntelliJ Idea to format the code for me.
> However I'll take another look.
>
> I wonder why the build failed though - since I made no logical changes to
> the code.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3068/files#r54874114>
> .
>